src/it/resources/com/google/checkstyle/test/chapter4formatting/rule411optionalbracesusage/InputUseOfOptionalBraces.java

```diff
@@ -71,26 +70,25 @@ public void testIf() {
     }
 
     // Invalid
+    if (condition()); // violation ''if' construct must use '{}'s.'
     if (condition()) // violation ''if' construct must use '{}'s.'
-      ;
-    if (condition()) // violation ''if' construct must use '{}'s.'
-    testIf();
+      testIf();
     if (condition()) // violation ''if' construct must use '{}'s.'
-    testIf();
+      testIf();
     else // violation ''else' construct must use '{}'s.'
-    testIf();
+      testIf();
     if (condition()) // violation ''if' construct must use '{}'s.'
-    testIf();
+      testIf();
     else {
       testIf();
     }
     if (condition()) {
       testIf();
     } else // violation ''else' construct must use '{}'s.'
-    testIf();
-    if (condition()) // violation ''if' construct must use '{}'s.'
-    if (condition()) // violation ''if' construct must use '{}'s.'
       testIf();
+    if (condition()) // violation ''if' construct must use '{}'s.'
+      if (condition()) // violation ''if' construct must use '{}'s.'
+          testIf();
```

Comment:
config was complaining about incorrected indented children, config was right here. Formatter didn't intended the children, I guess it got confused by the trailing comments